plugins/m/htmlsanity.py

```diff
@@ -635,10 +635,14 @@ def depart_definition_list(self, node):
 
     # no class="docutils" in <hr>
     def visit_transition(self, node):
-        self.body.append(self.emptytag(node, 'hr'))
+        if 'm-transition' in node['classes']:
+            self.body.append(self.starttag(node, 'p', ''))
```

Comment:
> (can't remember how to check, tho)

If I remember correctly, nodes have a `children` property, which is a list of nodes. I guess checking if it's empty or not could work.

I checked the class because it's the first thing that came to mind, to be honest.